pydriller/domain/commit.py

```diff
@@ -192,12 +190,18 @@ def __hash__(self):
     @property
     def source_code(self):
         warn('The use of `source_code` is deprecated. Use `content` instead.', DeprecationWarning, stacklevel=2)
-        return self.__source_code
+        if self.language_supported and type(self.content) == bytes:
+            return self.content.decode("utf-8", "ignore")
```

Comment:
Let's keep the try/except here, the old lines 751 to 757

Reply:
Done [e9be9d8](https://github.com/ishepard/pydriller/pull/231/commits/e9be9d807e656b51c682e9704639b0e3ee4c1853)